qa/L0_trace/test.sh

```diff
@@ -649,37 +647,109 @@ fi
 if [ `grep -c "\"log_frequency\":\"0\"" ./curl.out` != "1" ]; then
     RET=1
 fi
-if [ `grep -c "\"trace_file\":\"some_file.log\"" ./curl.out` != "1" ]; then
+if [ `grep -c "\"trace_file\":\"bls_trace.log\"" ./curl.out` != "1" ]; then
     RET=1
 fi
 
+set +e
+# Send bls requests to make sure simple model is traced
+for p in {1..4}; do
+    python -c 'import opentelemetry_unittest; \
+        opentelemetry_unittest.send_bls_request(model_name="ensemble_add_sub_int32_int32_int32")'  >> client_update.log 2>&1
+done
```

Comment:
I don't really understand this part - why isn't it somewhere in the python code, rather than being called from bash?

Reply:
That is to test that Triton Trace API logs bls request. I didn't want to write a separate client for this. `opentelemetry_unittest.py` tests OpenTelemetry Trace API.